com.servoy.extensions/src/com/servoy/extensions/plugins/rawSQL/RawSQLProvider.java

```diff
@@ -413,14 +413,15 @@ public boolean js_flushAllClientsCache(String serverName, String tableName)
 	 * //var action = SQL_ACTION_TYPES.INSERT_ACTION //pks inserted
 	 * //var action = SQL_ACTION_TYPES.UPDATE_ACTION //pks updates
 	 * var pksdataset = databaseManager.convertToDataSet(new Array(12,15,16,21))
-	 * var ok = plugins.rawSQL.notifyDataChange(databaseManager.getDataSourceServerName(controller.getDataSource()), 'employees', pksdataset,action)
+	 * var ok = plugins.rawSQL.notifyDataChange(databaseManager.getDataSourceServerName(controller.getDataSource()), 'employees', pksdataset,action,notifySelf)
 	 *
 	 * @param serverName
 	 * @param tableName
 	 * @param pksDataset
 	 * @param action
+	 * @param notifySelf
 	 */
-	public boolean js_notifyDataChange(String serverName, String tableName, IDataSet pksDataset, int action)
+	public boolean js_notifyDataChange(String serverName, String tableName, IDataSet pksDataset, int action, boolean notifySelf)
```

Comment:
we can't change an existing method by just adding a param, then all others calling the old one will be broken.
This should be an additional method where the old once calls the new one with "false"